manifests/fact.pp

```diff
@@ -13,7 +13,7 @@
   file { "${facterbasepath}/facts.d/${title}.yaml":
     ensure  => $ensure,
     owner   => 'root',
-    group   => 'puppet',
+    group   => 'root',
```

Comment:
I think we should base this on $::puppet::allinone instead of changing it to root for everyone.
We still have a lot of users that have not upgraded to puppet 4.